api/models/User.js

```diff
@@ -3,50 +3,20 @@ var promise       = require('promised-io/promise');
 var adBox         = require('boxfish-router');
 var colors        = require('colors');
 
+var customLogger = function customLogger() {
```

Comment:
I guess but do you reckon it's big enough to be its own module? I guess we
could use it as a helpers module

On Fri 12 Feb 2016 at 18:08, Daniel Oliveira <notifications@github.com>
wrote:

> In api/models/User.js
> <https://github.com/bxfsh/sails-hook-user/pull/4#discussion_r52776052>:
>
> > @@ -3,50 +3,20 @@ var promise       = require('promised-io/promise');
> >  var adBox         = require('boxfish-router');
> >  var colors        = require('colors');
> >
> > +var customLogger = function customLogger() {
>
> I feel like this should be in another file ( maybe another npm module ? )
> so ir can be used in all files across the hook
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bxfsh/sails-hook-user/pull/4/files#r52776052>.
>